### DISCUSSION TITLE
How to copy an entire stack of code in Android?

### DISCUSSION THREAD

**Discussion by kkhanmd** [pybricks/support#1448] [Q&A]
Is there a way to copy an entire stack of code on an Android tablet? (I do not have a mouse input, only touch control). When I try to press and hold on top of the stack only that top part of the stack comes for copy, is there a way to select the entire stack to copy? 
![Screenshot_20240216_160811_Chrome](https://github.com/pybricks/support/assets/160268632/ae3b6a9c-cfa4-483b-8164-4aeb6eec4fe3)

**Comment by BertLindeman** [comment]
Not sure if this works  for you. . . . 
Tried this:

press-long on an empty place on the canvas. 
Select ```Select all blocks```
Next press-long on any block and choose ```copy``` 
next press-long on the canvas and choose ```paste```

Is does not really what you want I think, but it maybe better than copying each and every block.
Hope this helps.

Bert
